pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -170,6 +160,10 @@ if [ "$dontLink" != 1 ]; then
     export NIX_LINK_TYPE_@suffixSalt@=$linkType
 fi
 
+if [ "$setClangTarget" == 1 ]; then
+    extraAfter+=(-target $clangTarget)
```

Comment:
@Mindavi ^

Reply:
:man_facepalming: great catch. I'll actually remove the old bit in the next push.

Reply:
Removed the old code: https://github.com/NixOS/nixpkgs/pull/176152/files#diff-ad7b67d4a92a2db43281c9c45aa74c32341a554fbca0b6b25be593f4648abb5eL300-L307

Reply:
Nice, now it makes more sense :)